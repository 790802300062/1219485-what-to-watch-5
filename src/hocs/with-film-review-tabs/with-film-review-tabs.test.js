import React from "react";
import renderer from "react-test-renderer";
import {MockComponent} from "../../test-data/test-data";
import withFilmReviewTabs from "./with-film-review-tabs";

const MockComponentWrapped = withFilmReviewTabs(MockComponent);
it(`renders withFilmReviewTabs`, () => {
  const tree = renderer
    .create(
        <MockComponentWrapped>
          <React.Fragment />
        </MockComponentWrapped>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
