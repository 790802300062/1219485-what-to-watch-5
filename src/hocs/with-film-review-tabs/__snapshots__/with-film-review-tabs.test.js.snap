// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders withFilmReviewTabs 1`] = `<div />`;
